src/main/java/org/fundacionjala/pivotal/pageobjects/login/SignInPage.java

```diff
@@ -84,6 +84,7 @@ public static Dashboard loginAs(final String username, final String password) {
      * @param password that.
      * @return a Dashboard object
      */
+    //TODO join both methods
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)